src/main/scala/ch/epfl/bluebrain/nexus/sbt/nexus/ReleasePlugin.scala

```diff
@@ -18,11 +18,38 @@ import sbtrelease._
   */
 object ReleasePlugin extends AutoPlugin {
 
+  lazy val buildInfoJson  = taskKey[File]("Generates build info json")
+
   override lazy val requires = sbtrelease.ReleasePlugin
 
   override lazy val trigger = allRequirements
 
   override lazy val projectSettings = Seq(
+    buildInfoJson := {
+      val file = target.value / "buildinfo.json"
+      val v = version.value
+      val major = v.split("\\.")(0)
+      val minor = v.split("\\.")(1)
+      val contents =
+        s"""{
+           |"name": "nexus-${name.value}",
+           |"version": "$v",
+           |"version_major": "$major",
+           |"version_minor": "$minor",
+           |"description"  : "${description.value}",
+           |"repository"   :  {
+           |    "url": "${homepage.value.map(_.toString).getOrElse("undefined")}",
+           |    "issuesurl": "${homepage.value.map(_.toString + "/issues").getOrElse("undefined")}"
+           |
+           |  },
+           |"license": "Apache License 2.0",
```

Comment:
The license is not always Apache 2. We need to pull this information from SBT. I would recommend doing the same for authors and contributors.